py/sample/test_mock_concurrent.py

```diff
@@ -1,8 +1,9 @@
 import pytest
 from pathlib import Path
 
-from serial_keel import connect
-from serial_keel_util import make_logger
+from serialkeel import connect
+from serialkeel.util import make_logger
```

Comment:
I don't think the `serial_keel_util.py` file has enough value to be kept.
Could you remove it?

Reply:
Yeah, if it's just that one function.

Reply:
Moved this function to serial_keel.py :+1: